readme.md

```diff
@@ -22,25 +25,94 @@ of the code between the tags for that feature.  Following that, you should remov
 `# START_FEATURE` and `# END_FEATURE` comments from the codebase.
 Following that, you should run `pip-compile`.
 
-The follwing is a list of tagged features in this repo:
+The following is a list of tagged features in this repo:
 
 ```
 bootstrap_messages
 django_social
 crispy_forms
+django_react
+debug_toolbar
+sentry
 ```
 
 The codebase also has a number of reference examples.  These are all marked with the comment:
 ```
 # TODO: delete me; this is just a reference example
 ```
 
+
+# Feature Descriptions
+
+
+## Django-React
+
+
+### Additional Setup
+
+
+When using this feature, make sure to install the Node.js requirements using the manager of your choice 
+(either `npm install` or `yarn install` will work) before proceeding with development.
+
+
+### Special Consideration for Running
+
+
+For development on localhost when using Django-React, you should run the following command in a separate terminal to
+your standard `runserver` command.
+    
+- `nwb serve --no-vendor`
+
+If you have configured everything correctly, you should see each command complete and notify you
+that the project is ready to be viewed.
```

Comment:
Add something like:

If you include `nwb` as a dependency, you can use the locally-installed nwb by running `node_modules/.bin/nwb server --no-vendor` instead of relying on a globally installed `nwb`.